fastly/block_fastly_service_v1_gcslogging.go

```diff
@@ -260,9 +261,16 @@ func (h *GCSLoggingServiceAttributeHandler) Register(s *schema.Resource) error {
 		blockAttributes["format"] = &schema.Schema{
 			Type:        schema.TypeString,
 			Optional:    true,
-			Default:     "%h %l %u %t %r %>s",
+			Default:     "%h %l %u %t \"%r\" %>s %b",
```

Comment:
Would it better to use backticks here? 

e.g.
```suggestion
			Default:     `%h %l %u %t "%r" %>s %b`,
```

Ps, consider this a suggestion everywhere else it is relevant in this PR.